pages/contact.tsx

```diff
@@ -0,0 +1,16 @@
+import Head from "next/head";
+import ContactPage from "@/components/general/ContactPage";
+
+export default function Contact() {
+  return (
+    <>
+      <Head>
+        <title>Contact</title>
+        <meta name="description" content="Generated by Taodemy" />
```

Comment:
Think about SEO

Reply:
Ok, I will follow the blog page to modify.